travharv/store.py

```diff
@@ -21,65 +22,13 @@
 DEFAULT_URN_BASE = "urn:traversal-harvesting:"
 
 
-class GraphConfigNameMapper:
-    """Helper class to convert config names into graph-names."""
-
-    def __init__(self, base: str = DEFAULT_URN_BASE) -> None:
-        """constructor
-
-        :param base: (optional) base_uri to apply,
-        - defaults to DEFAULT_URN_BASE = 'urn:traversal-harvesting:'
-        :type base: str
-        """
-        self._base = str(base)
-
-    def cfgname_to_ng(self, cfgname: str) -> str:
-        """converts local filename to a named_graph
-
-        :param cfgname: name of the config (trav-harv job context)
-        :type cfgname: str
-        :returns: urn representing the filename to be used as named-graph
-        :rtype: str
-        """
-        return f"{self._base}{quote(cfgname)}"
-
-    def ng_to_cfgname(self, ng: str) -> str:
-        """converts named_graph urn back into the local context name
-
-        :param ng: uri of the named-graph
-        :type ng: str
-        :returns: the name of the matchoing travharv config context
-        :rtype: str
-        """
-        assert ng.startswith(self._base), (
-            f"Unknown {ng=}. " f"It should start with {self._base=}"
-        )
-        lead: int = len(self._base)
-        return unquote(ng[lead:])
-
-    def get_cfgnames_in_store(self, store: RDFStore) -> Iterable[str]:
-        """selects those named graphs in the store.named_graphs under our base
-        and converts them into travharv config names
-
-        :param store: the store to grab & filter the named_graphs from
-        :type store: RDFStore
-        :returns: list of trvharv config names in that store
-        :rtype: List[str]
-        """
-        return [
-            self.ng_to_cfgname(ng)
-            for ng in store.named_graphs
-            if ng.startswith(self._base)
-        ]  # filter and convert the named_graphs to config names we handle
-
-
 class RDFStoreAccess(RDFStoreDecorator):
     """Decorator class adding some trav-harv specific features"""
 
     def __init__(
         self,
         core: RDFStore,
-        name_mapper: GraphConfigNameMapper = GraphConfigNameMapper(),
+        name_mapper: GraphNameMapper = GraphNameMapper(base=DEFAULT_URN_BASE),
```

Comment:
do we still need it here?
rdf_store has this built in nowadays...
refactoring to reuse should reflect what happened to syncfs 

see
https://github.com/vliz-be-opsci/py-sync-fs-triples/commit/b161ba40a9b85b9bc136fc5059d696ae3baee85f